bots/npm-update

```diff
@@ -112,15 +112,14 @@ def run(package, verbose=False, **kwargs):
         execute("npm", "upgrade", "--save", package)
 
         # Check if that did an upgrade
-        update = package_json()
-        after = update["dependencies"]
-        after[package] = after[package].strip(" ^~=<>")
```

Comment:
To clarify: before it added the prefix to all packages, and *only* changed the one `package`. So the prefixes for all others ended up in the final package.json.

Now it keeps the original `before` and *only* operates on the specific `package`.